python/tvm/relay/transform.py

```diff
@@ -593,3 +593,30 @@ def PartialEvaluate():
         The registered pass that performs partial evaluation on an expression.
     """
     return _transform.PartialEvaluate()
+
+
+def QuantizeAnnotate():
```

Comment:
actually `quantize` is not a general purpose transform, should we put them under the namespace `transform` or just `relay.quantize`? @tqchen

Reply:
I had the same question above. I wanted to just leave them in `quantize.py`. But should we also not expose `QuantizeAnnotate` and `QuantizeRealize` in `transform.h`? We then can only use them through packedfunc when we want to get the pass. This looks against `transform` to me somehow.

Reply:
> actually `quantize` is not a general purpose transform, should we put them under the namespace `transform` or just `relay.quantize`? @tqchen

+1

Reply:
done